core/trino-main/src/main/java/io/trino/sql/planner/LocalDynamicFilterConsumer.java

```diff
@@ -248,4 +248,9 @@ public synchronized String toString()
                 .add("summaryDomains", summaryDomains)
                 .toString();
     }
+
+    private static long getRetainedSizeInBytes(TupleDomain<DynamicFilterId> summary)
```

Comment:
Commit message `Refactor LocalDynamicFilterConsumer` is too generic, maybe `Extract getRetainedSizeInBytes method in LocalDynamicFilterConsumer` ?